cdatransform/transform/gdclib.py

```diff
@@ -59,8 +59,12 @@ def get_entities(original):
         yield (sample, "sample", "Initial specimen", sample, original)
         for portion in sample.get("portions", []):
             yield (portion, "portion", sample.get("sample_id"), sample, original)
-            for aliquot in portion.get("aliquots", []):
-                yield (aliquot, "aliquot", portion.get("portion_id"), sample, original)
+            for slide in portion.get("slides", []):
+                yield (slide, "slide", portion.get("portion_id"), sample, original)
+            for analyte in portion.get("analytes", []):
+                yield (analyte, "analyte", portion.get("portion_id"), sample, original)
+                for aliquot in analyte.get("aliquots", []):
+                    yield (aliquot, "aliquot", analyte.get("analyte_id"), sample, original)
 
 
 def specimen_from_entity(entity, _type, parent_id, sample, case):
```

Comment:
@druth314 your adjustments seem great.
We just need a tiny fix down below on line 74: identifier value should be assigned to id and identifier itself should have its own value:
"id": entity.get(id_key),
"identifier": entity.get("submitter_id"),

Reply:
the identifier is supposed to be the identifier for the entity, not specifically for the research subject/patient. id is supposedly going to be a unique identifier for the CDA system and identifier is a codeable concept which will save the identifier of the entity with respect to which DC the entity is seen in. For instance for specimens the id can be simply the sample_id seen in GDC, whereas the identifier is a codeable concept that shows the sample_id in GDC and the sample_id in PDC

Reply:
> For instance for specimens the id can be simply the sample_id seen in GDC, whereas the identifier is a codeable concept that shows the sample_id in GDC and the sample_id in PDC
I agree with this, all I'm saying is that this should be reflected in the code.
The way it is now:
"identifier": "sample_id"
"id": None